template.yaml

```diff
@@ -399,3 +425,4 @@ Resources:
         VolumeType: gp2
       CognitoOptions:
         Enabled: false
+      Role: !Ref  SnapshotRole.Arn
```

Comment:
Perhaps you can try to put the actions in the properties about 20 lines up? Where we have defined "es:ESHttpHead" ect